docs/application/native/guides/ui/tizen-ws-shell.md

```diff
@@ -63,41 +63,40 @@ init(const char *name)
 
 ## Showing and Hiding Quickpanel Window
 
-To show and hide quickpanel window when your application's window is activated, call the following functions:
+To show or hide quickpanel window when your application's window is activated, call the following functions:
 
 ```
-   ...
-
    if (show)
      tzsh_quickpanel_show(tzsh_qp); /* To show quickpanel window */
    else
      tzsh_quickpanel_hide(tzsh_qp); /* To hide quickpanel window */
-
-   ...
 ```
 
 ## Getting Visible Status of Quickpanel Window
```

Comment:
@gl-lee kindly change to:
## Get Visiblility Status of Quickpanel Window

Reply:
I think "Visibility" is correct, not "Visiblility".

Reply:
ok. I've changed to "Get Visiblility Status of Quickpanel Window".